lib/main_publiccloud.pm

```diff
@@ -95,6 +100,11 @@ sub load_latest_publiccloud_tests {
         }
         elsif (get_var('PUBLIC_CLOUD_CONTAINERS')) {
             load_container_tests();
+        } elsif (get_var('PUBLIC_CLOUD_SMOKETEST')) {
+            loadtest "publiccloud/smoketest";
+            loadtest "publiccloud/sev" if (get_var('PUBLIC_CLOUD_CONFIDENTIAL_VM'));
+            loadtest "publiccloud/xen" if (get_var('PUBLIC_CLOUD_XEN'));
+            loadtest "publiccloud/az_l8s_nvme" if (get_var('PUBLIC_CLOUD_INSTANCE_TYPE'));
```

Comment:
I don't understand this condition.

Reply:
Good catch, this is wrong. Let me fix this.

Reply:
Fixed, please mark as resolved if you're happy with the solution.